From bb8848ea424f9f8044b626d1c6e9c9b2803f98f0 Mon Sep 17 00:00:00 2001
From: Wally Gleemer <Wally.Gleemer@windriver.com>
Date: Tue, 25 Mar 2008 16:21:33 -0700
Subject: [PATCH] cavium: idle->thread_info no longer exists

Use task_thread_info(idle) since idle->thread_info no longer exists.

Signed-off-by: Wally Gleemer <Wally.Gleemer@windriver.com>
---
 arch/mips/cavium-octeon/smp.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/mips/cavium-octeon/smp.c b/arch/mips/cavium-octeon/smp.c
index 32a911f..eb646c9 100644
--- a/arch/mips/cavium-octeon/smp.c
+++ b/arch/mips/cavium-octeon/smp.c
@@ -84,7 +84,7 @@ void prom_boot_secondary(int cpu, struct task_struct *idle)
 	       cpu_logical_map(cpu));
 
 	octeon_processor_sp = __KSTK_TOS(idle);
-	octeon_processor_gp = (unsigned long) idle->thread_info;
+	octeon_processor_gp = (unsigned long)(task_thread_info(idle));
 	__sync();		/* Use sync so all ops are done. This makes the
 				   cycle counter propagate in a more bounded
 				   amount of time */
-- 
1.5.5.1

